libqtile/backend/x11/core.py

```diff
@@ -315,13 +314,6 @@ def _xpoll(self) -> None:
                     if not ret:
                         break
 
-            # Catch some bad X exceptions. Since X is event based, race
```

Comment:
wait what :D

Reply:
yeah don't mind it :) it's just spitting out garbage sometimes, I'm gonna take a look at what it did the following days.

Reply:
You were talking about the block comment tho or what? cause I can't figure it out.

Reply:
Yeah the fact that it seems to remove comments. Also when it simplifies code to a one-liner the comments are gone :(

Reply:
Yeah, @elParaguayo saw that too. I'll look into it.

Reply:
Thanks for doing this :)